test/integration/default/controls/packages_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# Overide by OS
+package_name = 'bash'
+package_name = 'cronie' if (os[:name] == 'centos') && os[:release].start_with?('6')
+
+control 'apache package' do
+  title 'should be installed'
+
+  package_name =
+    case platform[:family]
+    when 'debian', 'suse'
+      'apache2'
+    when 'redhat', 'fedora'
+      'httpd'
+    when 'arch'
```

Comment:
As discussed in the #formulas channel yesterday, `arch` is under the `linux` family:

```suggestion
    # `linux` here is sufficient for `arch`
    when 'linux'
```